texar/torch/data/tokenizers/sentencepiece_tokenizer.py

```diff
@@ -232,6 +235,7 @@ def default_hparams() -> Dict[str, Any]:
           loaded from the vocabulary file. In this case, all other
           configurations in `hparams` are ignored.
         * Otherwise, the tokenizer is trained based on `hparams['text_file']`.
```

Comment:
In this case, will the tokenizer automatically trained, or users need to call `tokenizer.train()` to train the tokenizer? Pls make this clearer in the doc